### DISCUSSION TITLE
How to generate msi file in preferred location in file explorer

### DISCUSSION THREAD

**Discussion by sansa20** [wixtoolset/issues#7460] [Questions]
I have a WixSetup project which is generating the msi file under the project bin folder in file explorer.
Is there any way I can change this and get the msi file to be generated at another preferred location on file explorer?
I tried unloading the project and editing the .wixproj code to achieve this but I keep getting an error.
I changed the value of the OutputPath and IntermediatePath to my preferred location.

![image](https://user-images.githubusercontent.com/64676335/235623915-4c36aff7-cf37-482b-a14a-6de23fd01c0a.png)

**Comment by BMurri** [comment]
Visual Studio requires that changes to IntermediatePath and OutputPath (or any of their related properties) are made inside of PropertyGroups that are conditioned by the "current" Configuration/Platform settings. This has been discussed in all the various places people go to complain about Visual Studio for years.
